### PR TITLE
Adds blueprinter 0.25.3 to gemfile

### PR DESCRIPTION
**Pull request by abrayard**
[sc-77]

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ gem 'hanami-model', '~> 1.3'
 gem 'pg'
 gem 'rack'
 gem 'shrine'
+gem 'blueprinter'
```
Comment (parkersprouse) [inline Gemfile:11]: I have to see if there's a cop for this, but we should keep dependencies in alphabetical order

File: README.md [hunk 2]
```diff
@@ -19,6 +19,18 @@ While using the Docker container for development is not required, it is highly r
...
 <br>
 
+Before running a `make init`, I had to run the following to get `pg` to install properly:
+
+<br>
+
+```
+brew install libpq
+export PATH="/opt/homebrew/opt/libpq/bin:$PATH"
+gem install pg
```
Comment (parkersprouse) [inline README.md:29]: I wonder if this is an M1 thing because I did not have this issue. Did this fix the nokogiri issue?

Comment (abrayard) [reply README.md:29]: Nope

File: Gemfile (hunk 1 above)
Comment (abrayard) [reply Gemfile:11]: Kk, I'll change it for now.

Comment (abrayard) [reply Gemfile]: `Bundler/OrderedGems`

**Review by parkersprouse — APPROVED** [review]

**Pull request merged**